README.html

```diff
@@ -0,0 +1,151 @@
+                    <li>None</li>
+                </ul>
+            </li>
+            <li>Technologies used
+                <ul>
+                    <li>None</li>
+                </ul>
+            </li>
+        </ul>
+
+        <p><br/></p>
+
+        <h2>Live Demo</h2>
+
+        <p><a href="https://projects.microverse.datanet-pt.net/Hello-Microverse/">Live Demo Link</a></p>
+
+        <p><br/></p>
+
+        <h2>Getting Started</h2>
+
+        <blockquote>
+            <p>This project can be run either from a local copy of the files (in it's directory structure) or by accessing it through a WEB server if it is installed as content for one.</p>
+
+            <p>To get a local copy up and running follow these simple example steps.</p>
+        </blockquote>
+
+        <p><br/></p>
+
+        <h3>Prerequisites</h3>
+
+        <blockquote>
+            <p><strong>NONE</strong></p>
+        </blockquote>
+
+        <h3>Setup</h3>
+
+        <blockquote>
+            <p><strong>NONE</strong></p>
+        </blockquote>
+
+        <h3>Install</h3>
+
+        <blockquote>
+            <p><strong><em>git clone https://github.com/Microverse-ajgomes/Hello-Microverse.git</em></strong></p>
+        </blockquote>
+
+        <h3>Usage</h3>
+
+        <blockquote>
+            <p><strong>Just access the index.html file through the browser or access the webserver where it has been deployed through the browser</strong></p>
+        </blockquote>
+
+        <h3>Run tests</h3>
+
+        <blockquote>
+            <p><strong>Not applyable</strong></p>
+        </blockquote>
+
+        <h3>Deployment</h3>
+
+        <blockquote>
+            <p><strong>Not applyable</strong></p>
+        </blockquote>
+
+        <p><br/></p>
+
+        <h2>Authors</h2>
+
+        <p>&#x1f464; <strong>Ant&oacute;nio Jos&eacute; Gomes</strong></p>
+
+        <ul>
+            <li>GitHub: <a href="https://github.com/ajose1024">@ajose1024</a></li>
+            <li>Twitter: <a href="https://twitter.com/ajose_1024">@ajose_1024</a></li>
+            <li>LinkedIn: <a href="https://linkedin.com/in/ajffg1024">LinkedIn</a></li>
+        </ul>
+
+        <p><br/></p>
+
+        <h2>&#x1f91d; Contributing</h2>
+
+        <p>Contributions, issues, and feature requests are welcome!</p>
+
+        <p>Feel free to check the issues page<!--<a href="../../issues/">issues page</a>-->.</p>
+
+        <p><br/></p>
+
+        <h2>Show your support</h2>
+
+        <blockquote>
+            <p>Give a &#x2B50; if you like this project!</p>
+        </blockquote>
+
+        <p><br/></p>
+
+        <h2>Acknowledgments</h2>
+
+        <ul>
+            <li>Hat tip to anyone whose code was used</li>
+            <li>Inspiration</li>
+            <li>etc</li>
+        </ul>
+
+        <p><br/></p>
+
+        <h2>&#x1F4DD; License</h2>
+
+        <p>This project is <a href="https://projects.microverse.datanet-pt.net/Hello-Microverse/MIT.html">MIT</a> licensed.</p>
+
+    </body>
+</html>
+
+
+
+
+
```

Comment:
[OPTIONAL]

- [ ] You can remove README.html file as well.
**README file always should have `md` termination, so make sure to not discard README.md**

Reply:
Although this is an exercise, this is a web content.
I have setup an actual web space for deployment of this "application" (https://projects.microverse.datanet-pt.net/Hello-Microverse), and this file exists so the README has a valid html form to be visible by the browser. It is accessed at:  https://projects.microverse.datanet-pt.net/Hello-Microverse/README.html
I have setup this web space as I DO TEST it in an actual webserver, to which content is feeded by a git clone of the repository (as stated in the README.md file)